pkg/sql/opt/xform/testdata/external/tpcc-no-stats

```diff
@@ -891,24 +891,23 @@ project
            ├── cardinality: [0 - 10]
            ├── key: (12)
            ├── fd: ()-->(11,13,21), (12)-->(14-18)
-           ├── index-join order
+           ├── scan order
            │    ├── columns: o_id:11!null o_d_id:12!null o_w_id:13!null o_c_id:14 o_entry_d:15 o_carrier_id:16 o_ol_cnt:17 o_all_local:18
+           │    ├── constraint: /13/12/-11
+           │    │    ├── [/10/1/2167 - /10/1/2167]
+           │    │    ├── [/10/2/2167 - /10/2/2167]
+           │    │    ├── [/10/3/2167 - /10/3/2167]
+           │    │    ├── [/10/4/2167 - /10/4/2167]
+           │    │    ├── [/10/5/2167 - /10/5/2167]
+           │    │    ├── [/10/6/2167 - /10/6/2167]
+           │    │    ├── [/10/7/2167 - /10/7/2167]
+           │    │    ├── [/10/8/2167 - /10/8/2167]
+           │    │    ├── [/10/9/2167 - /10/9/2167]
+           │    │    └── [/10/10/2167 - /10/10/2167]
```

Comment:
This one is cool. The plan for tpcc-no-stats now matches the plan for tpcc with stats. I think that's a good thing.